Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -0,0 +1,60 @@
+id: 89e6adbd-612c-4fbe-bc3d-32f81baf3b6c
+name: Azure DevOps Administrator Group Monitoring
+description: |
+  'This detection monitors for additions to project or project collection administration groups in an Azure DevOps Organization.'
+severity: High
+queryFrequency: 2h
+queryPeriod: 4h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+  - Escalation
+query: |
+
+  // Monitor for Project Administrator adds to *any* project
+  let MontiorAllProjects = false;
+  // If MonitorAllProjects is false, trigger only on Project Administrator add for the following projects
+  let ProjectsToMonitor = datatable(ProjectName:string)
+  [
+  'ProjectA',
+  'ProjectC'
+  ];
+  AzureDevOpsAuditing
```

Comment:
Generally we like to have a time frame specific for when this is used outside of the UX.  Recommend you add in | where TimeGenerated >= ago(timeframe) and add in a parameter at the top - let timeframe = 4h;

Reply:
fixed in PR